src/app/utilities/metadata/metadata.ts

```diff
@@ -0,0 +1,213 @@
+import { HOST_URL, META_TYPE } from '../constants';
+import {
+  KEYWORDS_ABOUT,
+  KEYWORDS_ADVISORY_CONSULTING,
+  KEYWORDS_ARTICLES,
+  KEYWORDS_BLOGS,
+  KEYWORDS_COACHING,
+  KEYWORDS_CONTACT,
+  KEYWORDS_HOME,
+  KEYWORDS_LOGIN,
+  KEYWORDS_NETWORKING,
+  KEYWORDS_NEURODIVERGENT_MATES,
+  KEYWORDS_NEURODIVERSITY_TRAINING,
+  KEYWORDS_PLACEMENTS,
+  KEYWORDS_SIGNUP,
+} from './keywords';
+
+export const metadata = {
+  home: {
+    title: 'Homepage - Neurodiversity Academy',
+    description: 'Homepage for Neurodiversity Academy',
+    images: [
+      {
+        url: '',
+      },
+    ],
```

Comment:
Check if we can use a `null`, `undefined` or if needed, an empty array. Apply to all of them.